core/src/main/java/io/confluent/rest/Application.java

```diff
@@ -507,6 +520,27 @@ protected void registerExceptionMappers(Configurable<?> config, T restConfig) {
     config.register(new GenericExceptionMapper(restConfig));
   }
 
+  /**
+   * Register header filter to ServletContextHandler.
+   * @param context The serverlet context handler
+   */
+  protected void configureHttpResponsHeaderFilter(ServletContextHandler context) {
+    String headerConfig = config.getString(RestConfig.RESPONSE_HTTP_HEADERS_CONFIG);
+    log.debug(headerConfig);
+    String[] configs = StringUtil.csvSplit(headerConfig);
+    Arrays.stream(configs)
+            .filter(config -> !config.trim().isEmpty())
+            .forEach(config -> RestConfig.validateHttpResponseHeaderConfig(config));
+    headerConfig = Arrays.stream(configs)
```

Comment:
It looks like the idea here is that you split apart the original `headerConfig`, validate that each header is valid, and then put the string back together. Can you skip putting it back together and just use the original value?

Reply:
Yes. We will treat case that ",   ," as invalid config, so simplify code as you suggested.